addons/l10n_us/views/res_config_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_account_config_settings_inherit">
```

Comment:
change id as per new guidelines... . "account_confirg_settings_inherit_form_view" something like that.
and keep id before model......... ;)